pkg/urbit/daemon/main.c

```diff
@@ -677,6 +708,11 @@ main(c3_i   argc,
   }
   // printf("vere: hostname is %s\n", u3_Host.ops_u.nam_c);
 
+  char tmp_cert_file_name[64];
+  memset(tmp_cert_file_name, 0, sizeof(tmp_cert_file_name));
+  strcpy(tmp_cert_file_name,"/tmp/urbit-ca-cert-XXXXXX");
+  u3_setup_cert_store(tmp_cert_file_name);
```

Comment:
It seems a little odd to specify the name here, with the extra copy, only for it be mutated in the function. But that's nitpicky ...

Reply:
Resolved as part of the exit(0) fix above.